src/widgets/SymbolOverview.tsx

```diff
@@ -1,3 +1,142 @@
-export const SymbolOverview = () => {
-	return null;
-};
\ No newline at end of file
+import React, {
+	useEffect, useRef, useState, type CSSProperties 
+} from "react";
+import { injectTradingView } from "../utils/injectTradingView";
+import { getId } from "../utils/widgetUtils";
+
+interface MediumWidgetBaseOptions {
```

Comment:
might be cleaner to move these interfaces and enums somewhere, unless you're running into typing issues.